.github/workflows/bdd.yaml

```diff
@@ -0,0 +1,30 @@
+name: BDD tests
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+
+jobs:
+  bdd:
+    runs-on: ubuntu-20.04
+    strategy:
+      matrix:
+        go-version:
+          - "1.20"
```

Comment:
please use version 1.18 (same as Dockerfile) for consistency between repositories